schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(40),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BIT,
```

Comment:
- Hey, the datatype for the `neutered` is `boolean` and not a string since we basically want to ascertain between a `TRUE` || `FALSE` which in our case is what we want to achieve. Please, kindly update this.
Furthermore, A `bit` only stores the `numbers 0 and 1` (or `null`) while A `boolean` only stores `true` and `false` (or `null`). Therefore,  A `number (0, 1)` is not a boolean, and a boolean value can be used anywhere a boolean expression is expected which is not the same for a bit.